### PR TITLE
feat: make it possible to choose image name

### PR DESCRIPTION
**Pull request by tmlye**
Extends the `svcs image-name` command to be able to specify a custom image name instead of using the folder name

### CODE REVIEW

File: src/commands/services.rs [hunk 1]
```diff
@@ -291,7 +291,7 @@ impl ManifestBuilder {
 
     pub fn get_image(mut self) -> Result<Self> {
         self.manifest.service.image =
-            get_image_name(&self.api_client, &self.token, &self.org_name, &None)?;
+            get_image_name(&self.api_client, &self.token, &self.org_name, &None, &None)?;
```
Comment (bittermandel) [inline src/commands/services.rs:294]: We should probably rename `get_image` and `get_image_name`. It returns name, but also has a parameter called `name`.

File: src/config/user.rs [hunk 2]
```diff
@@ -43,7 +43,6 @@ impl Token {
 }
 
 pub struct UserConfigLoader {
-    pub path: Utf8PathBuf,
```
Comment (bittermandel) [inline src/config/user.rs:46]: This wasn't used before?

**Review by bittermandel — APPROVED** [review]

File: src/config/user.rs (hunk 2 above)
Comment (tmlye) [reply src/config/user.rs:46]: no

File: src/commands/services.rs (hunk 1 above)
Comment (tmlye) [reply src/commands/services.rs:294]: the manifest already has a get_image function so I think it would have conflicts

**Pull request merged**